airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -34,6 +34,9 @@ class MyFacebookAdsApi(FacebookAdsApi):
 
     MAX_RATE, MAX_PAUSE_INTERVAL = (95, pendulum.duration(minutes=10))
     MIN_RATE, MIN_PAUSE_INTERVAL = (85, pendulum.duration(minutes=2))
+    MAX_SLEEP_TIME_FOR_RATE_LIMIT = 3600  # 1 hour
+    current_sleep_time = 0
```

Comment:
is this a constant or should it be a object field?

Reply:
It's a constant but because `MIN_RATE, MAX_RATE` etc are also defined same way above I followed the same for consistency. Can extract outside as well.